components/icon/stories/icon.stories.js

```diff
@@ -98,6 +88,34 @@ export default {
 export const Default = IconGroup.bind({});
```

Comment:
Now that the icons are loading so much faster, do we want to surface the workflow icons as the landing Storybook page for icons? They're lovely and it seems like the most common use-case.

Reply:
I'm not sure about that; instead of "Docs"? That doesn't seem consistent with the other components but I might be misunderstanding the suggestion.

Reply:
Not instead of docs, just thinking instead of a single icon for the Default page, we could show all the workflow icons instead.